python/plugins/db_manager/dlg_sql_window.py

```diff
@@ -91,10 +91,8 @@ def __init__(self, iface, db, parent=None):
         self.updatePresetsCombobox()
 
         self.geomCombo.setEditable(True)
-        self.geomCombo.lineEdit().setReadOnly(True)
 
         self.uniqueCombo.setEditable(True)
-        self.uniqueCombo.lineEdit().setReadOnly(True)
```

Comment:
I am not sure to measure the importance of such a modification ... is it really a problem to let the user enter its own column name if he is not happy with the column names given by the combo box? What should possibly happen ? A wrong column name ? Well, it is up to the user to know what he types ...